{
    "type": "bundle",
    "id": "bundle--5206f7f1-1650-4c86-b00c-55542e242b3b",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--13d2c501-9234-4bc9-9f1c-c0c38e3622ee",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2024-02-29T00:27:16.941274Z",
            "modified": "2024-02-29T00:27:16.941274Z",
            "name": "CVE-2020-36778",
            "description": "In the Linux kernel, the following vulnerability has been resolved:\n\ni2c: xiic: fix reference leak when pm_runtime_get_sync fails\n\nThe PM reference count is not expected to be incremented on\nreturn in xiic_xfer and xiic_i2c_remove.\n\nHowever, pm_runtime_get_sync will increment the PM reference\ncount even failed. Forgetting to putting operation will result\nin a reference leak here.\n\nReplace it with pm_runtime_resume_and_get to keep usage\ncounter balanced.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2020-36778"
                }
            ]
        }
    ]
}